src/Symfony/Component/Serializer/Normalizer/FlattenExceptionNormalizer.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes FlattenException instances.
+ *
+ * @author Pascal Luna <skalpa@zetareticuli.org>
+ *
+ * @experimental
```

Comment:
this doesn't work, it should be removed before merging
why was it addded?

Reply:
Hi @nicolas-grekas , just removed the experimental tag.

I also added a project where you can reproduce this exact bug:  https://github.com/monteiro/serializer-pr37087